examples/kitchen-sink/patches/@remix-run+dev+1.4.3.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/@remix-run/dev/compiler.js b/node_modules/@remix-run/dev/compiler.js
+index 902b251..d5b4240 100644
+--- a/node_modules/@remix-run/dev/compiler.js
++++ b/node_modules/@remix-run/dev/compiler.js
+@@ -211,6 +211,10 @@ async function watch(config$1, {
+     toWatch.push(config$1.serverEntryPoint);
+   }
+ 
++  if(config$1.serverDependenciesToBundle) {
```

Comment:
Based on this [comment](https://github.com/remix-run/remix/issues/1193#issuecomment-1063143520).